text/src/autogluon/text/text_prediction/utils.py

```diff
@@ -38,8 +39,8 @@ def parallel_transform(df, chunk_processor,
         return chunk_processor(df)
     else:
         chunks = np.array_split(df, num_process * 2)
-        with mp.Pool(num_process) as pool:
-            out_l = pool.map(chunk_processor, chunks)
+        with ThreadPoolExecutor(num_process) as pool:
+            out_l = list(pool.map(chunk_processor, chunks))
```

Comment:
What is performance impact? If it is a slowdown we should add code comment for why we do this